src/docs/deployment/android.md

```diff
@@ -1,107 +1,106 @@
-or by using the **Run** and **Debug**
-options in your IDE. By default,
-Flutter builds a _debug_ version of your app.
-
-When you're ready to prepare a _release_ version of your app,
-for example to [publish to the Google Play Store][play],
-this page can help. Before publishing,
-you might want to put some finishing touches on your app.
-This page covers the following topics:
-
-* [Adding a launcher icon](#adding-a-launcher-icon)
-* [Signing the app](#signing-the-app)
-* [Shrinking your code with R8](#shrinking-your-code-with-r8)
-* [Reviewing the app manifest](#reviewing-the-app-manifest)
-* [Reviewing the build configuration](#reviewing-the-build-configuration)
-* [Building the app for release](#building-the-app-for-release)
-* [Publishing to the Google Play Store](#publishing-to-the-google-play-store)
-* [Updating the app's version number](#updating-the-apps-version-number)
-* [Android release FAQ](#android-release-faq)
-
-## Adding a launcher icon
-
-When a new Flutter app is created, it has a default launcher icon.
-To customize this icon, you might want to check out the
-[flutter_launcher_icons][] package.
-
-Alternatively, you can do it manually using the following steps:
-
-1. Review the [Material Design product
-   icons][launchericons] guidelines for icon design.
-
-1. In the `<app dir>/android/app/src/main/res/` directory,
-   place your icon files in folders named using
-   [configuration qualifiers][].
-   The default `mipmap-` folders demonstrate the correct
-   naming convention.
-
-1. In `AndroidManifest.xml`, update the
-   [`application`][applicationtag] tag's `android:icon`
-   attribute to reference icons from the previous
-   step (for example,
+Thông thường khi phát triển,
+bạn có thể chạy thử ứng dụng của mình bằng dòng lệnh `flutter run` ở cửa sổ *command line*,
+hoặc bằng lựa chọn **Run** và **Debug**
+ở IDE của mình. Theo mặc định,
+Flutter sẽ xây dựng ra (*build*) một phiên bản _debug_ (*gỡ lỗi*) cho ứng dụng của bạn.
+
+Khi bạn đã sẵn sàng chuẩn bị một phiên bản _phát hành_ cho ứng dụng của mình,
+ví dụ như [phát hành lên Google Play Store][play],
+trang web này có thể cho bạn thêm thông tin. Trước khi phát hành,
+có thể bạn sẽ muốn chỉnh sửa một số chi tiết để ứng dụng hoàn thiện hơn.
+Dưới đây là một số hướng dẫn:
+
+* [Thêm icon biểu tượng cho ứng dụng](#adding-a-launcher-icon)
+* [Đăng kí ứng dụng](#signing-the-app)
+* [Rút gọn code với R8](#shrinking-your-code-with-r8)
+* [Kiểm tra bảng kê khai ứng dụng (app manifest)](#reviewing-the-app-manifest)
+* [Xem lại cấu hình xây dựng (build) (build configuration)](#reviewing-the-build-configuration)
+* [Xây dựng (build) ứng dụng để sẵn sàng phát hành](#building-the-app-for-release)
+* [Phát hành ứng dụng lên Google Play Store](#publishing-to-the-google-play-store)
+* [Cập nhật đánh số phiên bản của ứng dụng](#updating-the-apps-version-number)
+* [Các câu hỏi thường gặp khi phát hành ứng dụng Android](#android-release-faq)
+
+## Thêm icon biểu tượng cho ứng dụng
```

Comment:
Launcher = Trình khởi chạy ?

Reply:
Should we use either icon or biểu tượng?

Reply:
Em nghĩ để icon dễ hiểu hơn ạ, từ icon cũng khá phổ biến và là từ trong docs gốc